devel/jsonnet/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        google jsonnet 0.17.0 v
+revision            0
+categories          devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         The data templating language
+
+long_description    A data templating language for app and tool developers
+
+homepage            http://jsonnet.org/
+
+checksums           rmd160  e43a5cee00aaea3782407c9ce0b87d28f34fc3b9 \
+                    sha256  9f4450ff784952a0f5c593ce7dabd83cbda2eed62a4b01cc2ba507b32483c8dd \
+                    size    21859800
+
+use_configure       no
```

Comment:
this port is not using TheRightCompiler. Ideally you would use the `makefile` PG or perhaps switch to CMake. In anycase, the port in its current form does not build for me locally.

Reply:
@reneeotten Thanks for all these corrections.

I'm trying to get `jsonnet` to compile with cmake. It hits a link error with `ld: symbol(s) not found`.

It looks like there's an extra directory `${cmake.build_dir}/lib` with some compiled libraries.

I can't tell if this is a bug in the cmake build, or I just need to somehow add a `-L${cmake.build_dir}/lib` to the link command.

How do I do this in MacPorts?  If I do this, the cmake configure phase fails because the directory doesn't yet exist:
```
configure.ldflags-append \
                    ${cmake.build_dir}/lib

```

Failing this, it looks like an upstream issue.

Reply:
P.S. See 
[main.log](https://github.com/macports/macports-ports/files/5982275/main.log)

Reply:
Got this working per makefile as you describe below. Upstream's cmake appears pretty broken in several places.

Done.